src/libs/AttachmentModalHandler/index.ios.ts

```diff
@@ -0,0 +1,12 @@
+import {InteractionManager} from 'react-native';
+import type AttachmentModalHandler from './types';
+
+const attachmentModalHandler: AttachmentModalHandler = {
+    handleModalClose: (onCloseCallback) => {
+        InteractionManager.runAfterInteractions(() => {
```

Comment:
Can we please get a comment here explaining why this is needed only on iOS? 🙏

Reply:
This happens only on iOS native because its native modal animations run concurrently with Animated.FlatList, leading to conflicts. iOS enforces stricter rendering and animation synchronization, exposing timing issues.

Reply:
I love that explanation! Sorry I wasn't super clear, but can you please add that comment in the code so other people can easily see the explanation in the future? 🙏